src/bun.js/test/jest.zig

```diff
@@ -35,8 +35,6 @@ const getAllocator = @import("../base.zig").getAllocator;
 const JSPrivateDataPtr = @import("../base.zig").JSPrivateDataPtr;
 const GetJSPrivateData = @import("../base.zig").GetJSPrivateData;
 
-const default_timeout = std.time.ms_per_min * 5;
```

Comment:
Changed from `std.time.ms_per_min` to `std.time.ms_per_s`. This appeared to be a bug, since the default should be 5 seconds, not 5 minutes.